src/store/global/global.thunks.js

```diff
@@ -22,7 +22,7 @@ function setHermezEnvironment () {
     hermezjs.TxPool.initializeTransactionPool()
 
     if (!window.ethereum) {
-      return dispatch(globalActions.loadEthereumNetworkFailure(LOAD_ETHEREUM_NETWORK_ERROR.METAMASK_NOT_INSTALLED))
+      return dispatch(globalActions.loadEthereumNetworkSuccess({ name: 'mainnet' }))
```

Comment:
I maybe would dispatch a `{ name: undefined }` from here and not render the network name if you are using wallet connect, as i think that you can use any testnet also to connect to the app right? Or am I wrong?